tests/nnet/activations/test_elu.py

```diff
@@ -44,8 +39,8 @@ def test_elu_fwd():
     mygrad_func=elu,
     true_func=_np_elu,
     num_arrays=1,
-    assumptions=_finite_params,
-    kwargs={"alpha": lambda x: finite_floats | finite_floats.map(np.array)},
+    index_to_bnds={0: (-np.log(sys.float_info.max) / 100, np.log(sys.float_info.max) / 100)},
+    kwargs={"alpha": lambda x: _reasonable_floats | _reasonable_floats.map(np.array) | _reasonable_floats.map(Tensor)},
```

Comment:
do we have a black config? this line is 118 - default black will reformat but my work black uses 120